production/kubernetes/agent-traces.yaml

```diff
@@ -37,45 +37,45 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: grafana-agent-traces
-  namespace: YOUR_NAMESPACE
+  namespace: ${NAMESPACE}
 ---
 apiVersion: v1
 kind: Service
 metadata:
   labels:
     name: grafana-agent-traces
   name: grafana-agent-traces
-  namespace: YOUR_NAMESPACE
+  namespace: ${NAMESPACE}
 spec:
   ports:
-  - name: agent-http-metrics
-    port: 8080
-    targetPort: 8080
-  - name: agent-thrift-compact
+  - name: grafana-agent-traces-http-metrics
+    port: 80
+    targetPort: 80
+  - name: grafana-agent-traces-thrift-compact
```

Comment:
We should test this to make sure it's not too long, port names have a fixed length limit

Reply:
just tested - all good. limit is 63 chars